src/Nethermind/Nethermind.Network/TrustedNodes/TrustedNodesManager.cs

```diff
@@ -23,6 +24,7 @@ public class TrustedNodesManager : ITrustedNodesManager
         private ConcurrentDictionary<PublicKey, NetworkNode> _nodes = new();
         private readonly string _trustedNodesPath;
         private readonly ILogger _logger;
+        private readonly Channel<Node> _nodeChannel = Channel.CreateUnbounded<Node>();
```

Comment:
I think we need bounded channel, otherwise potential spam could take down the node? 2^16 limit looks ok to me.